apollo-router/src/cache/redis.rs

```diff
@@ -562,16 +562,29 @@ impl RedisCacheStorage {
     }
 
     pub(crate) async fn delete<K: KeyType>(&self, keys: Vec<RedisKey<K>>) -> Option<u32> {
-        self.inner
-            .del(keys)
-            .await
-            .map_err(|e| {
-                if !e.is_not_found() {
+        let mut h: HashMap<u16, Vec<String>> = HashMap::new();
```

Comment:
```suggestion
        let mut h: HashMap<u16, Vec<String>> = HashMap::with_capacity(keys.len());
```

Reply:
Does it make sense ?

Reply:
yes

Reply:
kinda

Reply:
there will be less entries in the hashmap than the number of keys

Reply:
Feel free to decide whatever is better. It was just a suggestion :)